Makefile

```diff
@@ -10,20 +10,70 @@ CHART_VERSION?=$(subst v,,$(GIT_TAG))
 KUBE_VERSION?="v1.22.7"
 CLUSTER_NAME?="operator-e2e"
 RAWCOMMITDATE=$(shell git log -n1 --format="%at")
-COMMITDATE?=$(shell date -d @"${RAWCOMMITDATE}" "+%FT%TZ")
+
+ifeq ($(shell go env GOOS),darwin) # Use the darwin/amd64 binary until an arm64 version is available
+	KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path --arch amd64 $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
+	COMMITDATE?=$(shell gdate -d @"${RAWCOMMITDATE}" "+%FT%TZ")
+else
+	KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path $(KUBEBUILDER_ENVTEST_KUBERNETES_VERSION))
+	COMMITDATE?=$(shell date -d @"${RAWCOMMITDATE}" "+%FT%TZ")
+endif
 
 LDFLAGS := -w -s
 LDFLAGS += -X "github.com/rancher/elemental-operator/pkg/version.Version=${GIT_TAG}"
 LDFLAGS += -X "github.com/rancher/elemental-operator/pkg/version.Commit=${GIT_COMMIT}"
 LDFLAGS += -X "github.com/rancher/elemental-operator/pkg/version.CommitDate=${COMMITDATE}"
 
+# Directories
+BIN_DIR := bin
+TOOLS_DIR := hack/tools
+TOOLS_BIN_DIR := $(abspath $(TOOLS_DIR)/$(BIN_DIR))
```

Comment:
Can this be simplified so we only use one parameter for the tools_bin_dir instead of 3? Just have it on one dir that can be overriden instead of having three (BIN_DIR, TOOLS_DIR and TOOLS_BIN_DIR)